middle_end/flambda2/parser/flambda_to_fexpr.ml

```diff
@@ -988,6 +992,7 @@ let bind_all_code_ids env unit =
   !env
 
 let conv flambda_unit =
+  Format.eprintf "%a\n%!" Flambda_unit.print flambda_unit;
```

Comment:
I'm guessing this wasn't meant to be committed?

Reply:
Fixed